README.md

```diff
@@ -36,8 +36,8 @@ namespace ConsoleApp
         {
             if (args.Length == 3)
             {
-                Uri accountUri = new Uri(args[0]);     // Account URL, for example: https://fabrikam.visualstudio.com                
-                String personalAccessToken = args[1];  // See https://www.visualstudio.com/docs/integrate/get-started/authentication/pats                
+                Uri accountUri = new Uri(args[0]);     // Account URL, for example: https://dev.azure.com/fabrikam                
+                String personalAccessToken = args[1];  // See https://docs.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/pats                
```

Comment:
minor, but the `en-us` should be removed from the url

Reply:
Thanks, fixed

Reply:
Thanks Paul, I missed this PR and just made the changes myself in a different PR. I am going to cancel this one